docs/administrator-handbook.md

```diff
@@ -94,6 +94,15 @@ that there are not other pending changes:
 ```
 # rpm-ostree install -yA <pkg>
 ```
+### Override rpm files
+
+The install --force-replacefiles feature introduces the ability to override files 
+from one RPM package with files from another RPM package.
+
+To utilize this feature, use the rpm-ostree install --force-replacefiles command. 
```

Comment:
I think this sentence is redundant - "utilize" and "use" are two ways of saying the same thing, and "the rpm-ostree install --force-replacefiles command" is what is demonstrated in the code below.

Reply:
No problem ill drop it!